eoscostarica/eos-rate
# fix(enviroment): solve dev env variables

### GH Issue

Resolves #627

### Steps to test

1. Make run
1. Make fresh with bad endpoint
1. Check hapi docker-compose variables are same than staging-envarioment
